lib/me.robot

```diff
@@ -0,0 +1,30 @@
+*** Comments ***
+@@ -0,0 +1,23 @@
+
+
+*** Settings ***
+Library     BuiltIn
+Library     Process
+
+
+*** Keywords ***
+Check ME State
+    [Documentation]    Keyword Checks The ME PCI Register Offset 0x40 To Find Current ME State, Returns:
+    ...    -Disabled (Soft) For 3
+    ...    -Disabled (HAP) for 2
+    ...    -Enabled for 0
+    TRY
+        ${out}=    Execute Command In Terminal    setpci -s 16.0 40.L
+        ${char}=    Evaluate    ${out}[3]
+        # Current Operation Mode: bits 16:19 of register HFSTS1 (0x40)
+    EXCEPT    Evaluating expression 'p' failed: NameError: name 'p' is not defined nor importable as module
```

Comment:
What exactly is this message?

Reply:
now i see making 3 different branches was a mistake .. here is the context: https://github.com/Dasharo/open-source-firmware-validation/pull/375#discussion_r1679311540